manifests/init.pp

```diff
@@ -104,7 +104,18 @@
     }
   }
 
-  package { 'rubygem-foreman_scap_client': ensure => $ensure, install_options => $install_options, } ->
+  $scappackage = $operatingsystem ? {
+    ubuntu => rubygem-foreman-scap-client,
```

Comment:
Debian (and thus Ubuntu) conventions say it should be `ruby-foreman-scap-client`.